boltdesignsystem/bolt
# DS-173: Remove Blueprints

## Jira

https://pegadigitalit.atlassian.net/browse/DS-173
https://pegadigitalit.atlassian.net/browse/DS-88

## Summary

Remove Blueprints NPM package, move files into Pages within Pattern Lab.

## Details

We're no longer supporting Blueprints, so I'm removing the Blueprints NPM package and all mentions of it in the codebase. The templates within Blueprints are being moved into Pages > Academy.

This PR is ready for review, **but not ready to merge**. We must wait for Blueprints to be removed from Academy ([ACD-101](https://pegadigitalit.atlassian.net/browse/ACD-101)) before we merge this work.

Changes include:
- Add `acd-` namespace to Academy-specific components
- Move "03-blueprints" to Pages and rename to "06-academy"
- Remove `@bolt/blueprints` from build, workspaces, remove the package.json itself.
- Prefix Academy pages with `acd-` to avoid naming conflicts with other templates.
- Rename Academy sub-components so that they don't appear in menu.

## How to test

- Run branch locally. You will need to run `yarn setup`.
- Verify Academy pages load properly, now located under Pages > Academy.

## Release notes

The `@bolt/blueprints` NPM package has been removed. Templates previously found in Blueprints are now in Pages > Academy. `@bolt/blueprints` has been removed as a dependency from the Academy project.
